Makefile

```diff
@@ -27,3 +27,11 @@ backstopjs-reference:
 
 backstopjs-test:
 	docker-compose exec backstopjs backstop test
+
+setup:
+	cp docker-compose.override.yml.dist docker-compose.override.yml
+	cp web/sites/default/example.settings.local.php web/sites/default/settings.local.php
+	docker-compose up -d
+	docker-compose exec -T php composer install
+	scripts/reload-local.sh
```

Comment:
According to the changes proposed [here](https://github.com/Metadrop/drupal-boilerplate/pull/46/files#r644306565), we could specify the site in the reload local script.
```
	scripts/reload-local.sh --site=$SITE
```